tensorrt_llm/_torch/modules/dwdp/weight_buffer.py

```
@@ -0,0 +1,742 @@
+                    shape=spec.full_shape,
+                    dtype=spec.dtype,
+                    device_id=self._device_id,
+                )
+
+            except Exception:
+                # Best-effort cleanup on error
+                for va, size in all_mappings:
+                    try:
+                        unmap_va(va, size)
+                    except Exception:
+                        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)